### ISSUE TITLE
Custom module is not displaying the output of the command, but displayed the command itself 

### ISSUE THREAD

**Issue by bluebyt** [JakeStanger/ironbar#482]
**Describe the bug**
I have a custom module that suppose to output the nb of update package in Archlinux, but I see the command itself in the bar.
 
config.corn:

```corn
    $pacman = {
        type = "custom"
        bar = [ 
            {
                type = "label"
                name = "pacman"
                class = "pacman"
                label = "  {{90000:checkupdates | wc -l}} packages"               
            }
        ]
    }
```

**Expected behavior**
The bar should output the result. e.g.  "  5 packages"

**System information:**
 - Distro: [Arch Linux]
- Compositor: [Wayfire]
 - Ironbar version: [0.12.0]

**Configuration**
[config.corn](https://pastebin.com/RsMpq5h3)
[style.css](https://pastebin.com/zB6Jq0pG)

**Screenshots**
![20240304_070249](https://github.com/JakeStanger/ironbar/assets/18442224/0272999f-4c3a-4380-bd25-bf8530b5dba4)

**Comment by JakeStanger** [comment]
Ironbar v0.12 is from May last year, and there have been a few fixes to the script parser since then. Can you try updating the latest version and see if the issue still occurs first please?

**Comment by bluebyt** [comment]
I am already with latest git version of Ironbar.
ironbar -v                         
2024-03-04T13:36:11.223869Z  INFO ironbar: 49: Ironbar version 0.12.0
2024-03-04T13:36:11.223890Z  INFO ironbar: 50: Starting application
2024-03-04T13:36:11.226705Z  INFO ironbar: 120: Shutting down

**Comment by JakeStanger** [comment]
The latest released version is 0.14.1. Git is on 0.15.0-pre

**Comment by bluebyt** [comment]
Ok I found the problem, when I updated I forgot to do the last command:
install target/release/ironbar ~/.local/bin/ironbar
Now the version is 0.15.0-pre
Now everything is working as expected.
Thanks for your quick response!

![20240304_093437](https://github.com/JakeStanger/ironbar/assets/18442224/1e14adaf-c3f4-4116-9b22-9f066b7299c8)

I will close this.
